src/main/java/org/kiwiproject/jersey/client/ServiceIdentifier.java

```diff
@@ -0,0 +1,129 @@
+package org.kiwiproject.jersey.client;
+
+import static java.util.Objects.isNull;
+import static org.kiwiproject.base.KiwiPreconditions.checkArgumentNotNull;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.time.Duration;
+
+@SuppressWarnings("WeakerAccess")
+@NoArgsConstructor
+@Getter
+@Setter
```

Comment:
Do we allow setters in order to support external configuration e.g. from a YAML file?

Reply:
I'm not sure, but I don't like it :-)